app/services/GetBusinessDetailsService.scala

```diff
@@ -34,6 +35,9 @@ class GetBusinessDetailsService @Inject()(val getBusinessDetailsConnector: GetBu
       case success: IncomeSourceDetailsModel =>
         logger.debug(s"[getBusinessDetailsService][getBusinessDetails] - Retrieved Get Business Details:\n\n$success")
         success
+      case error: IncomeSourceDetailsError if error.status == NOT_FOUND =>
+        logger.warn(s"[getBusinessDetailsService][getgetBusinessDetails] - Retrieved Income Source Details:\n\n$error")
```

Comment:
pls extend IncomeSourceDetailsResponseModel with ~NotFound version and update log description with message like: Income Source Details not found as this is really not and error as state in the story"

Reply:
Done